azure-nextgen-ts-webapp-privateendpoint-vnet-injection/index.ts

```diff
@@ -0,0 +1,150 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+import * as network from "@pulumi/azure-nextgen/network/latest";
+import * as resources from "@pulumi/azure-nextgen/resources/latest";
+import * as web from "@pulumi/azure-nextgen/web/latest";
+import * as pulumi from "@pulumi/pulumi";
+import * as random from "@pulumi/random";
+
+const config = new pulumi.Config();
+
+// setup a resource group
+const resourceGroupName = config.get("resourceGroupName") || "webapp-pvtendpoint-vnet-injection";
+const location = config.get("location") || "westus2";
+const resourceGroup = new resources.ResourceGroup("resourceGroup", {
+    resourceGroupName: resourceGroupName,
+    location: location,
+});
+
+const serverfarm = new web.AppServicePlan("serverfarm", {
+    kind: "app",
+    location: location,
```

Comment:
If you update to 0.6.1, you can omit `location` property everywhere by setting the `azure-nextgen:location` config value.

Reply:
Done